usr/share/rear/conf/default.conf

```diff
@@ -3541,6 +3559,25 @@ EFI_STUB_EFIBOOTMGR_ARGS=
 # see https://github.com/rear/rear/pull/1182#issuecomment-282252770
 REAR_INITRD_COMPRESSION=""
 
+##
+# USE_RAMDISK
+#
+# Configure rescue system to create and pivot to a ramdisk instead of initramfs.
+#
+# This is required by some backup software that wants to check for actually available disk space.
+# Examples are: GALAXY11
```

Comment:
According to the removed obsolete ramdisk scripts for GALAXY via
https://github.com/rear/rear/pull/2937/files/549cb99adb2566725310fbcfab6ec74c061fbde1..15b763af73e7ade480c6e226062bdb59efad0e01
it seems all GALAXY versions we support in ReaR need USE_RAMDISK so
```
Examples are: BACKUP=GALAXY GALAXY7 GALAXY10 GALAXY11
```

Reply:
Good catch!